x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/get_baseline_and_deviation_rates.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { LOG_RATE_ANALYSIS_TYPE } from '@kbn/aiops-log-rate-analysis';
+
+export function getLogRateChange(
+  analysisType: typeof LOG_RATE_ANALYSIS_TYPE[keyof typeof LOG_RATE_ANALYSIS_TYPE],
+  baselineBucketRate: number,
+  deviationBucketRate: number
+) {
+  const logRateChange =
+    baselineBucketRate > 0
+      ? analysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+        ? `${Math.round((deviationBucketRate / baselineBucketRate) * 100) / 100}x higher`
+        : `${Math.round((baselineBucketRate / deviationBucketRate) * 100) / 100}x lower`
+      : analysisType === LOG_RATE_ANALYSIS_TYPE.SPIKE
+      ? i18n.translate(
+          'xpack.aiops.logRateAnalysis.resultsTableGroups.logRateIncreaseLabelColumnTooltip',
+          {
+            defaultMessage: '{deviationBucketRate} docs from 0 in baseline',
+            values: { deviationBucketRate },
+          }
+        )
+      : i18n.translate(
+          'xpack.aiops.logRateAnalysis.resultsTableGroups.logRateDecreaseLabelColumnTooltip',
+          {
+            defaultMessage: '0 docs from ${baselineBucketRate} in baseline',
```

Comment:
Need to remove the `$` here, but looks like there is something wrong with the logic as `baselineBucketRate` for a dip is showing as `0` in my tests:

<img width="240" alt="Screenshot 2024-06-19 at 17 00 21" src="https://github.com/elastic/kibana/assets/7405507/d7cbb8c3-132c-49e2-b6eb-258591e780f6">

Reply:
Fixed the left in $ from the string interpolation in ff861eb3f280d3812b7536fcbabcd06eced039cc
Looking into the logic now 👍 Good catch!

Reply:
Fixed the calculation in 2686407c0070e1c6d9024c2df9462397f91f4253